Multisig/UI/App/UIKit/UI Library/UIFont+Styles.swift

```diff
@@ -95,15 +95,15 @@ extension GNOButtonStyle {
     static let plain = GNOButtonStyle(appearance: [
         (.normal, GNOButtonAppearance(backgroundImage: nil, textAttributes: [
             .foregroundColor: #colorLiteral(red: 0, green: 0.5490000248, blue: 0.451000005, alpha: 1),
-            .font: UIFont.gnoFont(forTextStyle: .footnote2)
```

Comment:
This will break the style of "Learn more" button in "TransactionsListConflictHeaderTableViewCell"

Reply:
How to fix that button, because by changing this in your previous PR, all the buttons in the app changed

Reply:
I'll update